executor/executor.go

```diff
@@ -192,6 +192,11 @@ func (e *baseExecutor) Next(ctx context.Context, req *chunk.Chunk) error {
 	return nil
 }
 
+func (e *baseExecutor) updateDeltaForTableID(id int64) {
```

Comment:
Mutex is needed in `UpdateDeltaForTable`.

Reply:
There is already mutex inside `txnCtx.UpdateDeltaForTable`.